### ISSUE TITLE
Update version to 1.0.0?

### ISSUE THREAD

**Issue by mazswojejzony** [ekryski/launchpad#55]
Rationale: due to the way npm's [caret ranges](https://docs.npmjs.com/misc/semver#caret-ranges-1-2-3-0-2-5-0-0-4) work it is not easy to update `launchpad` in projects using it. 

E.g. `wct-local` has `launchpad` as its dependency with version defined as `^0.4.4.`. So in order to update `launchpad` to `0.5.0` (latest release as of Oct 23, 2015) one needs to update deps on `wct-local` manually.

Alternative solution: re-release `0.5.0` as `0.4.10` though personally I'd prefer the `1.0.0` as `launchpad` seems to be production ready.

**Comment by daffl** [comment]
Yes, that dependency will have to be updated. For a 1.0 release I'd really like to see #11 though. We can probably now steal the flags from https://github.com/karma-runner/karma-chrome-launcher/blob/master/index.js#L34 and similar.

**Comment by mazswojejzony** [comment]
OK, I raised/updated relevant issues on both wct and wct-local to update the launchpad version.

**Comment by daffl** [comment]
I'll leave this issue open. Maybe that'll get me to finish the profile support ;)

**Comment by mazswojejzony** [comment]
:+1:
